### PR TITLE
Bump vault version to 2.5.0 [wip]

### PR DESCRIPTION
**Pull request by mprasil**
I've bumped Vault to 2.5.0 but didn't have a time to extensively test. There were some new features added, so it's possible some things might be broken.

### CODE REVIEW

**Comment by dani-garcia** [comment]
According to [OrganizationUserType.cs](https://github.com/bitwarden/core/blob/master/src/Core/Enums/OrganizationUserType.cs) the new manager role has a value of three, so we need to make sure that it doesn't break anything, as we do a lot of `thisrole > thatrole`.

Also, #242 should be fixed before we merge this, otherwise adding ciphers to organizations probably won't work.

**Comment by mprasil** [comment]
Yeah there's multiple things to fix before this can be merged to master. [I triggered docker image build from the branch](https://hub.docker.com/r/mprasil/bitwarden/builds/b3ctmsekiigttrjnqrqsmxd/), so that people can test what works and what is broken.

**Comment by mprasil** [comment]
Yeah, I'm afraid we will start checking for manager once we implement some manager-related functionality. But as you say, I don't see how we could make that better with diesel.

**Comment by mprasil** [comment]
Okay, I've fixed #242 and also improved the `PartialOrd` handling of unknown `i32` type. I think it should be ready to merge if you didn't spot any other issues.

File: src/db/models/organization.rs [hunk 1]
```diff
@@ -83,6 +83,21 @@ impl PartialOrd<i32> for UserOrgType {
         }
         return None
     }
+
+    fn gt(&self, other: &i32) -> bool {
+        match self.partial_cmp(other) {
+            Some(Ordering::Less) => false,
+            _ => true,
```
Comment (dani-garcia) [inline src/db/models/organization.rs]: This would return true when equal

Comment (mprasil) [reply src/db/models/organization.rs]: I.. ..totally.. uhm.. left that there to test if you read my code. Definitely not a mistake. :sweat_smile:

**Comment by mprasil** [comment]
Fixed the `gt()` bug. Well spotted!

**Comment by dani-garcia** [comment]
Alright, I made a new release with the vault 2.4.0, and now this is ready to merge.

**Comment by mqus** [comment]
@dani-garcia Could you please re-release 1.4.0 (or 1.4.1)? the changes of this PR were merged after the tagged commit...

**Comment by mprasil** [comment]
@mqus, that was the plan. 😄 We expect some bugs with Vault 2.5.0, so people wanting more stable experience will use 1.4.0 (and the appropriate docker image version)

If you want this code, use master and the latest docker image that should be ready in couple hours. (Currently in the pipe)

**Comment by mqus** [comment]
... I had somehow read that vault 2.5.0 support is in 1.4.0... sry, my fault (It's 2.4.0 (which was also in 1.3.0 already))

**Pull request merged**